lib/tasks/config.rake

```diff
@@ -27,45 +27,43 @@ namespace :config do
     label_template_config = YAML.load_file(Rails.root.join('config/label_templates.yml'))
 
     puts 'Fetching submission_templates...'
-    submission_templates = api.order_template.all.each_with_object({}) { |st, store| store[st.name] = st.uuid }
+    query = Sequencescape::Api::V2::SubmissionTemplate.select(:uuid, :name).paginate(per_page: 100)
+    submission_templates = Sequencescape::Api::V2.merge_page_results(query).to_h { |st| [st.name, st.uuid] }
 
     puts 'Fetching purposes...'
     query = Sequencescape::Api::V2::Purpose.select(:uuid, :name).paginate(per_page: 100)
     all_purposes = Sequencescape::Api::V2.merge_page_results(query).index_by(&:name)
 
     purpose_config =
       ConfigLoader::PurposesLoader.new.config.map do |name, options|
-        PurposeConfig.load(name, options, all_purposes, api, submission_templates, label_template_config)
+        PurposeConfig.load(name, options, all_purposes, submission_templates, label_template_config)
       end
 
     puts 'Preparing purposes...'
     tracked_purposes = purpose_config.map { |config| all_purposes[config.name] ||= config.register! }
 
     # Build the configuration file based on the server we are connected to.
     CONFIG = # rubocop:todo Lint/ConstantDefinitionInBlock
-      {}.tap do |configuration|  # rubocop:todo Metrics/BlockLength
+      {}.tap do |configuration|
+        # TODO: Y24-190 - Remove this once we have moved everything else over to V2;
```

Comment:
backlog story created for this? 
or comment no longer needed?

Reply:
This is referencing the EPIC that this issue is part of.  It will be gone by the end of the EPIC.